aviator.yml

```diff
@@ -0,0 +1,17 @@
+spruce:
+  - base: ci/infra/meta.yml
+    prune:
+      - meta
+    merge:
+      - with_in: ci/infra/
+        regexp: ".*yml"
+        except:
+          - credentials.yml.example
+          - credentials.yml
```

Comment:
considering we don't use cred files, this can be removed, might make people think they are missing the files locally

Reply:
Done 👍🏻